mongomock/aggregate.py

```diff
@@ -87,6 +87,9 @@
     '$toLower',
     '$toUpper',
 ]
+misc_operators = [
+    '$toString',
```

Comment:
Let's add it to the string operators (the string operations convert non-strings to string already).

Reply:
np